src/internal/connector/graph/errors.go

```diff
@@ -25,6 +25,10 @@ const (
 	errCodeSyncStateNotFound           = "SyncStateNotFound"
 	errCodeResourceNotFound            = "ResourceNotFound"
 	errCodeMailboxNotEnabledForRESTAPI = "MailboxNotEnabledForRESTAPI"
+	tooManyRequests                    = 429
+	serviceUnavailable                 = 503
+	gatewayTimeout                     = 504
+	internalServerError                = 500
```

Comment:
Golang's http library has const values for all of these.  We should use those, not declare our own.